mlflow/langchain/utils.py

```diff
@@ -280,14 +291,6 @@ def _validate_and_wrap_lc_model(lc_model, loader_fn):
         )
 
     if special_chain_info := _get_special_chain_info_or_none(lc_model):
-        if isinstance(lc_model, langchain.chains.RetrievalQA) and version.parse(
```

Comment:
note: This is unrelated to this change, but the minimum supported version is `0.0.224` now so this code should be cleaned up.